examples/webgl/src/main.rs

```diff
@@ -123,13 +104,25 @@ impl App {
 
         gl.draw_arrays(GL::TRIANGLES, 0, 6);
 
-        let handle = {
-            let link = link.clone();
-            request_animation_frame(move |time| link.send_message(Msg::Render(time)))
-        };
+        // Gloo-render's request_animation_frame has this extra closure 
+        // wrapping logic running every frame, unnecessary cost.
+        // Here constructing the wrapped closure just once.
+
+        let gl = gl.clone();
+
+        let f = Rc::new(RefCell::new(None));
+        let g = f.clone();
```

Comment:
More descriptive names would be nice for the example. Maybe `draw` and inline `g` completely:

```
let draw = Rc::new(RefCell::new(None));
*draw.clone().borrow_mut() = Some(...);
```

Reply:
Yes fair point thank you.